src/NUnit3TestAdapterInstallV2/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="NUnit.NUnit3TestAdapter" Version="4.0.0.0" Language="en-US" Publisher="Terje Sandstrom" />
+    <Identity Id="NUnit.NUnit3TestAdapter" Version="3.6.0.1" Language="en-US" Publisher="Terje Sandstrom" />
```

Comment:
* I need to update the assemblies too. 
* I have tested it now on VS 2015, and it works there.  I don't have the earlier versions on my machine here, but it working on 2015 is a good sign.  
* I see one file in the commit here I overlooked, and I dont recognize, the .vs/slnx.sqlite file, I don't think that should be included. 
* Where do you see a new install folder?  If there is a new one, that is not intentional.